docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -303,14 +303,14 @@ Fields:
     to project commit activity. To run something on a regular schedule
     regardless of commit activity, consider using [periodic builds](Project-and-Distro-Settings#periodic-builds)
     instead.
--   `task_group`: a [task
-    group](#task-groups)
+-   `task_group`: a [task group](#task-groups)
     may be defined directly inline or using YAML aliases on a build
     variant task. This is an alternative to referencing a task group
     defined in `task_groups` under the tasks of a given build variant.
 -   `tags`: optional list of tags to group the build variant for alias definitions (explained [here](#task-and-variant-tags))
--   Build variants support [all options that limit when a task will run](#limiting-when-a-task-will-run). If set for the
-    build variant, it will apply to all tasks under the build variant.
+-   Build variants support [all options that limit when a task will run](#limiting-when-a-task-will-run)
```

Comment:
This link (+ the other links to `limiting-when-a-task-will-run`) will break because of the new header title. Can you update all the links in this file to that header?

Reply:
True yes fixed!